src/Compiler/CodeGen/IlxGen.fs

```diff
@@ -9069,11 +9100,23 @@ and GenMethodForBinding
                 | [ h ] -> Some h
                 | _ -> None
 
-            let ilCodeLazy =
-                CodeGenMethodForExpr cenv mgbuf (tailCallInfo, mspec.Name, eenvForMeth, 0, selfValOpt, bodyExpr, sequel)
+            let ilLazyCode =
+                DelayCodeGenMethodForExpr
+                    cenv
+                    mgbuf
+                    (tailCallInfo,
+                     mspec.Name,
+                     eenvForMeth,
+                     //{ eenvForMeth with
+                     //    delayCodeGen = false
+                     //},
```

Comment:
Probably should remove it instead of just commenting?

Reply:
yes, thanks